syft/pkg/cataloger/java/archive_parser.go

```diff
@@ -16,7 +22,7 @@ import (
 	"github.com/anchore/syft/syft/pkg/cataloger/generic"
 )
 
-var _ generic.Parser = parseJavaArchive
+const MavenBaseURL = "https://repo1.maven.org/maven2"
```

Comment:
This could also be a java configuration to allow folks to configure additional repositories. (consider this an optional comment)

Reply:
never mind, we can always introduce this later

Reply:
updated to a configurable value with the above as default